### ISSUE TITLE
Feature Request: golemcli command to mass delete tasks

### ISSUE THREAD

**Issue by kascheri12** [golemfactory/clay#1648]
As my node has accumulated many tasks, there is a lot to cleanup sometimes. With task persistence coming really soon I'd definitely like a way to delete all tasks that are Timeout status or other statuses also.

**Comment by maaktweluit** [comment]
hey @kascheri12 

Great idea!
If you are on a linux system ( mac needs some gnu-tools ) you can use:
```
golemcli tasks show | grep "Waiting\|Completed" | grep -o '^\S*' | sed 's/^/golemcli tasks delete /' | bash -s
```

change `Waiting\|Completed` to the statuses you want to delete and it will clean them all up one by one. Each one separated by `\|`

If you want to test it first and see what it will do remove ` | bash -s` and you will see the commands it creates before executing them.

Poke me in #testers chat on rocket if you need some more help on this. Then we can leave the issue focussed on the golem feature

**Comment by maaktweluit** [comment]
Resolved by: #3303
